cuda/test/solver/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-ginkgo_create_test(lower_trs_kernels ADDITIONAL_INCLUDES ${CUDA_INCLUDE_DIRS})
-ginkgo_create_test(upper_trs_kernels ADDITIONAL_INCLUDES ${CUDA_INCLUDE_DIRS})
+ginkgo_create_cuda_test(lower_trs_kernels)
+ginkgo_create_cuda_test(upper_trs_kernels)
```

Comment:
they do not have any kernel needs to be compiled by nvcc

Reply:
see https://github.com/ginkgo-project/ginkgo/pull/1368#discussion_r1283361036

Reply:
I do not think it is necessary. but I do not have a strong opinion.